modules/services/servers/src/main/java/au/com/shiftyjelly/pocketcasts/servers/model/DiscoverModel.kt

```diff
@@ -62,7 +62,8 @@ data class DiscoverRow(
     @field:Json(name = "uuid") override val listUuid: String?,
     @field:Json(name = "regions") val regions: List<String>,
     @field:Json(name = "sponsored") val sponsored: Boolean = false,
-    @field:Json(name = "curated") override val curated: Boolean = false
+    @field:Json(name = "curated") override val curated: Boolean = false,
+    @field:Json(name = "sponsored_podcasts") val sponsoredPodcasts: List<SponsoredPodcast> = emptyList(),
```

Comment:
I like how you added the comma after the last list item so that if/when we add another element the diff will only be one line instead of two. 👍